packages/office-ui-fabric-react/src/components/Rating/Rating.tsx

```diff
@@ -44,86 +68,57 @@ export class Rating extends BaseComponent<IRatingProps, IRatingState> {
   }
 
   public render() {
-    let stars: JSX.Element[] = [];
-    for (let i = this.props.min as number; i <= (this.props.max as number); ++i) {
-      stars.push(this._renderStar(i));
+
+    let stars = [];
+
+    for (let i = this.props.min as number; i <= (this.props.max as number); i++) {
+      let ratingStarProps: IRatingStarProps = {
+        fillPercentage: this._getFillingPercentage(i),
+        disabled: this.props.disabled ? true : false
+      };
+
+      stars.push(
+        <button
+          className={ css('ms-Rating-button', styles.ratingButton, {
+            ['ms-Rating--large ' + styles.rootIsLarge]: this.props.size === RatingSize.Large,
+            ['ms-Rating--small ' + styles.rootIsSmall]: this.props.size !== RatingSize.Small
+          }) }
+          key={ i }
+          { ...((i === Math.ceil(this.state.rating as number)) ? { 'data-is-current': true } : {}) }
+          onFocus={ this._onFocus.bind(this, i) }
+          disabled={ this.props.disabled || this.props.readOnly ? true : false }
+          role='presentation'
+        >
+          { this._getLabel(i) }
+          < RatingStar key={ i + 'rating' }  {...ratingStarProps} />
+        </button >
+      );
     }
 
     return (
       <div
-        className={ css('ms-Rating', this.props.className, {
-          ['ms-Rating--large ' + styles.rootIsLarge]: this.props.size === RatingSize.Large
-        }) }
-        role='application'
+        className={ css('ms-Rating-star') }
+        aria-label={ this.props.disabled ? '' : this._getRatingComponentAriaLabel() }
       >
-        <div className={ css('ms-Rating-container', styles.container) } role='radiogroup' aria-labelledby={ this.props.ariaLabelId }>
+        <FocusZone
```

Comment:
how does this behave with mobile screen readers? will I be able to change the value?

Reply:
how to test this on mobile screen reader? I mean how can I host it on mobile app?

Reply:
you can host it on your local box and then open that site on a mobile device?